test/cookbooks/python-webapp-test/recipes/default.rb

```diff
@@ -15,3 +15,51 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+include_recipe 'yum-ius'
+include_recipe 'yum-epel'
+
+%w(gcc python27 python27-devel python27-pip).each do |pkg|
+  package pkg
+end
+
+group 'chef' do
+  action :create
+end
+
+user 'chef' do
+  action :create
+  gid 'chef'
+end
+
+path = '/opt/whats_fresh/'
+
+hash = {
+  'path' => path,
+  'engine' => 'django.db.backends.sqlite3',
+  'dbname' => "#{path}/yourdatabasename.db"
+}
+
+python_webapp 'whats_fresh' do
+  create_user true
```

Comment:
I prefer the 'treat the programmer as an adult' method and do not do anything unless explicitly instructed.